media/js/base/site.js

```diff
@@ -5,7 +5,56 @@
 (function () {
     'use strict';
     window.site = {
-        getPlatform: function (ua, pf) {
```

Comment:
check file once again

Reply:
@Aayushap1 please stop randomly approving pull requests, thank you